### ISSUE TITLE
100% false positives on `network_after_suspend_auto`

### ISSUE THREAD

**Issue by kissiel** [canonical/checkbox#992]
### Bug Description

The job `network_after_suspend_auto` always passes.
This is due to the job definition not having and interface **option** in the `command` field, and the logic of the script being wrong.

### To Reproduce

Launch any Checkbox.
Select SRU test plan.
Make sure the `suspend/network_after_suspend_auto` is selected.
Unplug the adapter.
Wait until the job runs.

### Environment

22.04 desktop.

### Relevant log output

```shell
========[ Running job 32 / 32. Estimated time left (at least): 0:00:20 ]========
---------------------[ suspend/network_after_suspend_auto ]---------------------
ID: com.canonical.certification::suspend/network_after_suspend_auto
Category: com.canonical.plainbox::suspend
... 8< -------------------------------------------------------------------------
disconnected
Interface "None" not found.
------------------------------------------------------------------------- >8 ---
Outcome: job passed
```
```


### Additional context

_No response_
